### DISCUSSION TITLE
Boilerplate merge to main branch?

### DISCUSSION THREAD

**Discussion by timougue** [NoQuarterTeam/boilerplate-remix#697] [General]
Hi Jack, 

thanks a lot for this boilerplate! By when are you thinking to merge from develop to main branch? I am asking since I am looking for a "mature/proven" boilerplate with the stack you are building. What are the remaining issues you want to improve in the next weeks?

Thx!
Tim

**Comment by JClackett** [comment]
Hey! We are actually thinking of moving to tailwind, as we have been having issues with chakra-ui, so will be working on my hat refactor soon
